src/lumigo_tracer/lumigo_utils.py

```diff
@@ -246,15 +246,16 @@ def establish_connection(host):
     return None
 
 
-def prepare_host(host):
+def get_edge_host(region: Optional[str] = None):
+    host = Configuration.host or EDGE_HOST.format(region=region or os.environ.get("AWS_REGION"))
```

Comment:
I think that we already do `os.environ.get("AWS_REGION")` somewhere, maybe extract it to a function